src/vs/workbench/contrib/terminal/common/terminal.ts

```diff
@@ -475,3 +475,43 @@ export const enum TERMINAL_COMMAND_ID {
 	NAVIGATION_MODE_FOCUS_NEXT = 'workbench.action.terminal.navigationModeFocusNext',
 	NAVIGATION_MODE_FOCUS_PREVIOUS = 'workbench.action.terminal.navigationModeFocusPrevious'
 }
+
+interface TerminalDataBuffer {
+	timeoutId: any;
```

Comment:
This should be `number` because it lives in `common/`

Reply:
If I use `number` here, it causes errors, because the return of `setTimeout` here is `NodeJS.Timer`

Reply:
The fix there is to use window.setTimeout instead (the browser one)

Reply:
If I try that, I get this error:
![image](https://user-images.githubusercontent.com/641685/66668711-d3a00180-ec23-11e9-9fa8-b898bb4bd9d6.png)

Reply:
Ah ok, I didn't think VS Code had the issue but I guess it does, `any` is fine.